.github/workflows/unit-tests.yml

```diff
@@ -68,6 +68,24 @@ jobs:
         run: |
           sudo apt-get update && sudo apt-get install libmysqlclient-dev libxmlsec1-dev lynx
 
+      # This is needed until the ENABLE_BLAKE2B_HASHING can be removed and we
+      # can stop using MD4 by default.
+      - name: enable md4 hashing in libssl
+        run: |
+          cat <<EOF | sudo tee /etc/ssl/openssl.cnf
+          [openssl_init]
+					providers = provider_sect
```

Comment:
tabs -> spaces

Reply:
Weirdly, converting these to spaces breaks the workflow, and I couldn't easilyfigure out why.  I meant to add a comment to that effect, I'll do so before merging this.

Reply:
Oh you know, it's because you're using a heredoc to write the literal contents of that block to openssl.cnf. I bet openssl.cnf syntax requires tabs instead of spaces, or something like that. There's a probably a prettier way to do it but I'm not really concerned. Thanks for the comment.